plugins/imudp/imudp.c

```diff
@@ -1355,6 +1372,8 @@ CODEmodInit_QueryRegCFSLineHdlr
 		addInstance, NULL, STD_LOADABLE_MODULE_ID));
 	CHKiRet(omsdRegCFSLineHdlr((uchar *)"udpserveraddress", 0, eCmdHdlrGetWord,
 		NULL, &cs.pszBindAddr, STD_LOADABLE_MODULE_ID));
+	CHKiRet(omsdRegCFSLineHdlr((uchar *)"udpservermulticastinterface", 0, eCmdHdlrGetWord,
```

Comment:
Please remove the old-style config directives. We do not want to add any more of them.